src/util/cnames.js

```diff
@@ -139,7 +139,13 @@ const generateCNAMEsFile = (cnames, file) => {
     for (const i in cnamesKeys) {
         const cname = cnamesKeys[i];
         const data = cnames[cname];
-        cnamesList.push(`  "${cname}": "${data.target}"${Number(i) === cnamesKeys.length - 1 ? '' : ','}${data.noCF ? ` ${data.noCF}` : ''}`)
+        
+        const target = data.target.split('.');
+        if(target[target.length - 1] === '') {
+            target.splice(-1, 1);
+        }
```

Comment:
Please do this where all the other modifications happen, in the parsing logic, not in the generation logic: https://github.com/js-org/js.org-cleanup/blob/25497f6457b0f7bd6b20d44631a3b3b054746ee9/src/util/cnames.js#L54-L93

Reply:
noted.